### ISSUE TITLE
Filtering the translations table

### ISSUE THREAD

**Issue by luukvdmeer** [ipeaGIT/gtfstools#46]
Probably very low priority, since I rarely see GTFS feeds with this table present, but just for the sake of completeness: as far as I could find the [translations table](https://developers.google.com/transit/gtfs/reference#translationstxt) is not filtered in any of the filter functions. This table contains translations from certain field values to other languages. If these fields are not present anymore after a filter operations (e.g. after filtering trips or routes), they are not needed anymore in the translations table.

**Comment by dhersz** [comment]
You are correct, the translations table is not currently filtered in any of the filter functions. I explicitly left it out because it is rarely used, as you mentioned, and also because the table is a bit more complicated to filter because of how it is structured.

As you correctly guessed it's not high priority in the moment, but thanks for opening the issue anyway! I'll leave it open and may tackle it someday. Also feel free to submit a PR.
